### DISCUSSION TITLE
listview update in widget

### DISCUSSION THREAD

**Discussion by lomesh-5686** [ABausG/home_widget#286] [Help and Q&A]
i can add listview in widget but not update instant. i'll remove widget and add again on screen then update data in widget. how to fix this issue ?

**Comment by NguyenDang4902** [comment]


**Reply by NguyenDang4902** [reply]
Just add the line below. After checking other projects using this package, I added it, and it works like a charm

                val serviceIntent = Intent(context, AppWidgetService::class.java).apply {
                    putExtra("widgetData", listDataString)
                    data = Uri.parse(toUri(Intent.URI_INTENT_SCHEME))  // Add this line
                }
